package.json

```diff
@@ -21,14 +21,13 @@
   "bugs": "https://github.com/MindscapeHQ/raygun4js/issues",
   "license": "SEE LICENSE IN https://github.com/MindscapeHQ/raygun4js/blob/master/LICENSE",
   "devDependencies": {
-    "@wdio/cli": "^8.36.1",
-    "@wdio/dot-reporter": "^8.36.1",
-    "@wdio/jasmine-framework": "^8.36.1",
-    "@wdio/local-runner": "^8.36.1",
-    "@wdio/selenium-standalone-service": "^8.14.0",
-    "@wdio/spec-reporter": "^8.36.1",
-    "@wdio/static-server-service": "^8.36.1",
-    "chromedriver": "^124.0.3",
+    "@wdio/cli": "^9.16.2",
+    "@wdio/dot-reporter": "^9.16.2",
+    "@wdio/jasmine-framework": "^9.16.2",
+    "@wdio/local-runner": "^9.16.2",
+    "@wdio/selenium-standalone-service": "^8.15.0",
```

Comment:
just checking if this is needed anymore?

I noticed most of the `wdio` jumped a major version but this remains at 8.x, I ran `npm view @wdio/selenium-standalone-service` and `8.15.0` is the latest version but there is a deprecated message. 

This deprecated message is a slightly misleading, but might be worth validating if the selenium-standalone-service is deprecated in favor of WebDriverIO v9's automatic driver management so could possibly be removed?